client/dashboard/data/me-profile.ts

```diff
@@ -1,6 +1,7 @@
 import wpcom from 'calypso/lib/wp';
 
 export interface UserProfile {
+	advertising_targeting_opt_out: boolean;
```

Comment:
Interesting... is this change actually being used in this PR?

Reply:
It's being used here https://github.com/Automattic/wp-calypso/pull/105263/files/46eb5536a2ab4578a76a4cab51162dd7dc633f48